nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2195,13 +2195,16 @@ private boolean mayBeNullExpr(VisitorState state, ExpressionTree expr) {
       return false;
     }
     // the logic here is to avoid doing dataflow analysis whenever possible
-    Symbol exprSymbol = ASTHelpers.getSymbol(expr);
-    boolean exprMayBeNull;
     switch (expr.getKind()) {
       case NULL_LITERAL:
         // obviously null
-        exprMayBeNull = true;
+        return true;
```

Comment:
doing this i think was the reason why i saw a small but noticeable difference in the `NullawayRelease` JMH benchmark, YMMV

Reply:
Yes, this is a good change I think.  I cannot imagine a handler wanting to override the nullness of `null` 🙂 Let's just add a comment stating we don't allow handlers to override, so we return early as a performance optimization?

Reply:
ok recommend adjusted... calling it a perf optimization felt redundant because we also dont do that for the other literals above. WDYT ?